overrides/browsers/chrome-override.json

```diff
@@ -1,3 +1,100 @@
 {
-    "features": { }
+    "features": {
+        "cookie": {
+            "settings": {
+                "trackerCookie": "enabled",
```

Comment:
This duplication is unfortunate I raised: https://app.asana.com/0/1200890834746050/1202348774384704/f